test/slow-down.test.js

```diff
@@ -0,0 +1,25 @@
+import { test } from 'tap'
+import Fastify from 'fastify'
+import slowDownPlugin from '../index.js'
+import { HEADERS } from '../lib/constants.js'
+
+test('Should start to delay the API', async t => {
+  // create the fastify server
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  fastify.get('/', async () => 'Hello from Fastify!')
+
+  let res = await fastify.inject('/')
```

Comment:
this and other tests will have to change because we don't want the default value of the max option to be 0, meaning, it's not a sensible default